dashboard/test/factories/factories.rb

```diff
@@ -365,7 +341,7 @@
     end
 
     trait :with_migrated_google_authentication_option do
-      after(:create) do |user|
+      after(:create_commit) do |user|
```

Comment:
:scream: I remember this being kind of a mess, but never dug into the root cause.  Nice investigation on this!